### ISSUE TITLE
`PAUSE` and `COMMIT` operations fail for the jobs with an autoscaled disk

### ISSUE THREAD

**Issue by sidoruka** [epam/cloud-pipeline#998]
Related to #837

`PAUSE` and `COMMIT` operations fail with the following error for the jobs with an autoscaled disk (observed in `release/0.15` only for now):

```
[ERROR] 2020-03-11 16:49:10.856 [https-jsse-nio-8080-exec-8] ExceptionHandlerAdvice - This operation has been aborted: uri=/pipeline/restapi/run/1111/commit/check;client=10.244.0.1;session=7CF499DB8923EE41187FAA5F620B744C;user=user1.
java.lang.NullPointerException: null
        at com.epam.pipeline.manager.cluster.performancemonitoring.CAdvisorMonitoringManager.fetchDiskUsage(CAdvisorMonitoringManager.java:268) ~[classes!/:0.15.0.4713.4453ee5841edbb5904481f81b824ae363d00271d]
        at com.epam.pipeline.manager.cluster.performancemonitoring.CAdvisorMonitoringManager.matchToMonitoringStats(CAdvisorMonitoringManager.java:239) ~[classes!/:0.15.0.4713.4453ee5841edbb5904481f81b824ae363d00271d]
        at com.epam.pipeline.manager.cluster.performancemonitoring.CAdvisorMonitoringManager.convertFromRawToMonitoringStatsList(CAdvisorMonitoringManager.java:206) ~[classes!/:0.15.0.4713.4453ee5841edbb5904481f81b824ae363d00271d]
        at com.epam.pipeline.manager.cluster.performancemonitoring.CAdvisorMonitoringManager.executeStatsRequest(CAdvisorMonitoringManager.java:170) ~[classes!/:0.15.0.4713.4453ee5841edbb5904481f81b824ae363d00271d]
        at com.epam.pipeline.manager.cluster.performancemonitoring.CAdvisorMonitoringManager.executeContainerRequest(CAdvisorMonitoringManager.java:158) ~[classes!/:0.15.0.4713.4453ee5841edbb5904481f81b824ae363d00271d]
```

**Comment by tcibinan** [comment]
Checked that `PAUSE/RESUME` and `COMMIT` works in version `0.16.0.4715.6d66122b4865148cccb4f14e6b8da54d8233d49f`.

**Comment by sidoruka** [comment]
Fixed in `release/0.15` (958a94cdf899f35118dfd4d4ebfe6a8986476082) and `develop` (11f562e66952623cc0d782e8129e4eace97b8745)

**Comment by tcibinan** [comment]
Just for historical purposes both releases `0.15` and `0.16` were affected but only `0.15` deployment used `cadvisor` as resources monitoring backend which ended up breaking `PAUSE\RESUME` and `COMMIT` features.
